packages/nimble-components/src/table/styles.ts

```diff
@@ -117,6 +118,25 @@ export const styles = css`
         position: absolute;
     }
 
+    .column-divider:hover {
+        border-color: ${borderHoverColor};
+    }
+
+    .column-divider.visible {
+        display: block;
+        z-index: ${ZIndexLevels.zIndex1};
+    }
+
+    .column-divider.active {
+        border-color: ${borderHoverColor};
+    }
+
+    .header-container:hover .column-divider.left,
+    .header-container:hover .column-divider.right {
```

Comment:
Can't `left` and `right` be omitted?:
```suggestion
    .header-container:hover .column-divider {
```

Reply:
Yes, it seems like they can be omitted. I'm not sure why the styling wasn't originally written that way.